components/script/dom/element.rs

```diff
@@ -3831,16 +3830,50 @@ impl VirtualMethods for Element {
     }
 }
 
-impl SelectorsElement for DomRoot<Element> {
+#[derive(Clone, PartialEq)]
+pub enum SelectorWrapper<'a> {
```

Comment:
This is necessary because of `selectors::OpaqueElement`? Would be good to document the fix.

Reply:
It's necessary because DomRoot isn't defined in the same crate.